index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+
+
+  <link href="https://fonts.googleapis.com/css?family=Source+Serif+Pro:700" rel="stylesheet">
+  <title>Responsive News Reader</title>
+</head>
+
+<body>
```

Comment:
It would be good to have indentation used here. It will make the HTML structure easier to read